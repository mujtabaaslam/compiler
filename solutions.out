===== test1 =====
Usage: my-project [flags] [args]
 Available flags:
  -length prints the lengths of each of the arguments
  -help  Display this list of options
  --help  Display this list of options
===== test2 =====
this
should
print
line
by
line
===== test3 =====
3
4
2
7
2
4
knfnewinap
