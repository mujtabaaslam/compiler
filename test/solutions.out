===== test1 =====
1
===== test2 =====
[(,if,(,3,<=,4,),then,(,if,true,then,1,else,2,),else,(,2,+,3,),)]
===== test3 =====
(if (3 <= 4) then (if true then 1 else 2) else (2 + 3))
===== test4 =====
0
===== test5 =====
[3,/,2,+,5,-,6,*,2]
===== test6 =====
3 / 2 + 5 - 6 * 2
===== test7 =====
2
===== test8 =====
[if,(,true,),then,2,else,3]
===== test9 =====
if (true) then 2 else 3
